README.MD

```diff
@@ -256,6 +387,24 @@ npm install
 react-native run-ios # or react-native run-android
 ```
 
+## Troubleshooting
+
+<details>
+    <summary>Running example</summary>
```

Comment:
I think this title is missing something. I would rather put `Problems running the example` or `I have problems running the example`. What do you think?

Reply:
`I have problems running the example` seems legit